app/assets/stylesheets/config/_colors.scss

```diff
@@ -241,3 +241,33 @@
   --reaction-comment-color: var(--accent-warning);
   --reaction-comment-bg: var(--accent-warning-a10);
 }
+
+:root {
```

Comment:
Adding these variables for new buttons (`.c-btn`) to not mess around with old ones (`.crayons-button`).  I put them in separate `:root` to actually differentiate old vs new - it should not cause any harm to performance or anything.

I'll soon kick off efforts to clean up our variables anyway.

Reply:
I think it's probably worth adding this context as a comment in this file too

Reply:
Good point!